.github/FUNDING.yml

```diff
@@ -0,0 +1,4 @@
+# These are supported funding model platforms
+
+patreon: ArthurMilchior
```

Comment:
I'm confused as to why you are putting your funding information into the module that I wrote and maintain.